docs/guides/layers.md

```diff
@@ -84,7 +84,18 @@ or 3 less than the total number of dimensions of the layer, allowing you to
 browse volumetric timeseries data and other high dimensional data. See for
 example these cells undergoing mitosis in this volumetric timeseries:
 
-![napari viewer with an image of a cell undergoing mitosis. The scroll bar below the canvas controls the timeseries, allowing different stages of mitosis to be visible.](./images/mitosis.webm)
+```{raw} html
```

Comment:
Not sure whats up with this one
I get
<img width="784" alt="image" src="https://github.com/napari/docs/assets/76622105/c417c10e-738e-4ca1-bacd-17681bed74dd">

Reply:
Can't get it to play in Safari or on phone Safari.